README.md

```diff
@@ -34,6 +34,12 @@ To install, *first modify the `values.yaml` file to add redis and mongo details,
 
 Follow the instructions in the Notes that follow the installation to install the controller for Service Mesh sidecar injection.
 
+## Installing TIB
+The Tyk Identity Broker (TIB) is a microservice portal that provides a bridge between various Identity Management Systems such as LDAP, Social OAuth (e.g. GPlus, Twitter, GitHub), legacy Basic Authentication providers, to your Tyk installation (https://tyk.io/docs/concepts/tyk-components/identity-broker/).
+
+Once you have installed `Gateway` and `Dashboard` component you can configure `tib.conf` and `profile.json` and use helm upgrade command to install TIB.
```

Comment:
Can you update the configurable values in `tib.conf` and `profile.json` ?
For example, tell about `GOOGLE-OAUTH-TOKEN`, `API-To-GRANT-ACCESS-TO` and so on. Create a table of items which needs to be configured. Else people might miss on the key/values to change.

Reply:
added GitHub link for configuring tyk, which contains all the required information